README.md

```diff
@@ -60,4 +82,6 @@ Give a ⭐️ if you like this project!
 
```

Comment:
## OPTIONAL

- [ ] Good job acknowledging the designer, Kindly add a link to the design on behance 👍🏿  [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)